Sources/SwiftTileserverCache/WebServer.swift

```diff
@@ -417,11 +417,15 @@ public class WebServer {
     }
     
     private func touch(fileName: String) {
+		let fileManager = FileManager()
+        if !fileManager.fileExists(atPath: fileName) {
+			Log.warning("Failed to touch \(fileName)")
+		}
         do {
             var url = URL(fileURLWithPath: fileName)
             var resourceValues = URLResourceValues()
             resourceValues.contentAccessDate = Date()
-            try url.setResourceValues(resourceValues)
+        //    try url.setResourceValues(resourceValues)
```

Comment:
Commending out the `setResourceValues `function that actually sets the resource access date on disk makes it so that the `touch `function does nothing.

Reply:
it does nothing now on certain setups that why it throws the error this is just a temp work around for whoever till you get it resolved ..... i tried to fix it but i couldn't so stopping the spam is good enough for now lol all the images are still generated, its because docker cant "touch" the eternal files from docker something isn't letting them

Reply:
Can you go into any of your cache folders and show me the output of this command `stat *`?

Because your MR literally just comments out the function that would set the access time. (but maybe it's set automatically)